gnosis/safe-react
# Fix crash with Coinbase Wallet after upgrade to latest version

## What it solves

Closes #2359 Crash when connecting with Coinbase Wallet

## How this PR fixes it
After the upgrade some naming broke the Coinbase configuration

## How to test it
Connect using Coinbase Wallet and check that the app is not breaking anymore. Signing and executing transactions work as expected
